server/routes/ad.ts

```diff
@@ -735,6 +737,45 @@ const getAnomalyResults = async (
   }
 };
 
+const matchDetector = async (
```

Comment:
just curious: what does this API do? search detector by name?

Reply:
basically returns `true` or `false` if the passed params match any existing detector name, without returning any extra detector information that a user may not have access to if using FGAC

Reply:
See backend PR for more https://github.com/opendistro-for-elasticsearch/anomaly-detection/pull/286